apptuit/apptuit_client.py

```diff
@@ -2,6 +2,7 @@
 Client module for Apptuit APIs
 """
 
+from os import environ
```

Comment:
import os
use os.environ wherever looking up the environment variables. 
Makes code easy to read.